src/main/java/mate/jdbc/service/DriverDaoService.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverDaoService {
+    Driver create(Driver driver);
```

Comment:
I would advise against using the title with DaoService , Dao its Dao. Service its Service